workshops/color_scheme_generator/README.md

```diff
@@ -0,0 +1,255 @@
+---
+name: "Color Scheme Generator"
+description: "Create color schemes using React Hooks"
+author: "@giridhar7632"
+---
+
+# Color Scheme Generator
+
+In this workshop we are going to create a colorful app - 'random-color scheme generator' using React and [React Hooks](https://reactjs.org/docs/hooks-intro.html). It looks something like this.
+
+![Final Result](https://cloud-16p9skn27.vercel.app/0colorscheme.png)
+
+Here's the [Live demo](https://color-scheme-generator.giridharhackclu.repl.co/) and [Source Code](https://repl.it/@Giridharhackclu/Color-scheme-generator#src/index.js)
+
+## Prerequisites
+
+Before starting this workshop, you should have some knowledge about [React.js](https://reactjs.org) and some [ES6](https://www.w3schools.com/Js/js_es6.asp) features of JavaScript.
+
+React Fundamentals
+ - [JSX](https://reactjs.org/docs/introducing-jsx.html)
+ - [Components and Props](https://reactjs.org/docs/components-and-props.html)
+ - [State and LifeCycle](https://reactjs.org/docs/state-and-lifecycle.html)
+
+If you are familiar with these, let’s get going.
+
+## React Hooks API
+
+[Hooks](https://reactjs.org/docs/hooks-intro.html) are introduced in React 16.8 version. Previously, [Class components](https://reactjs.org/docs/components-and-props.html#function-and-class-components) were the only way to define a component that had its own state, and lifecycle methods. [Functional components](https://reactjs.org/docs/components-and-props.html#function-and-class-components) of React, which are light and more flexible, are limited in functionality. Hooks enable us to use state and other React features, like lifecycle methods and event handlers, without writing a Class component.
+
+There are many benefits of using Functional components. They are :
+
+- Easier to read, test, and debug.
+- Have better performance.
+- More reusable.
+
+> **Note:** Hooks don’t work inside classes. But you can use them instead of writing classes.
+
+## Setup
+
+For setting up the project fork the repl [here](https://repl.it/@Giridharhackclu/color-scheme-generator-starter#README.md). The starter repl contains ['create-react-app'](https://github.com/facebook/create-react-app) installed. There are some basic styles applied, you will get cleared along with the workshop.
```

Comment:
@giridhar7632 please review the phrasing very closely, here's an example of a grammar / phrasing mistake.